src/pages/index.js

```diff
@@ -41,62 +177,145 @@ editProfileMOB.addEventListener("click", () => {
+    "#add-card-modal-submit-button"
+  );
+  submitButton.textContent = "Saving...";
+
+  api
+    .addNewCard(cardData)
+    .then((res) => {
+      addNewCard(res, "prepend");
+      addPlacePopup.popupForm.reset();
+      addPlaceFormValidator.toggleButtonState();
+      addPlacePopup.close();
+    })
+    .catch((err) => {
+      console.log(err);
+    })
+    .finally(() => {
+      submitButton.textContent = "Create";
+    });
 }
 
-const handleImageClick = (cardData) => {
-  maxImagePopup.open(cardData);
-};
+//Handles card deletion
+let cardtoDeleteData;
 
-//***************************************************************************************************************************//
-//                                                     Objects                                                               //
-//***************************************************************************************************************************//
+//Handles confirm delete and sets delete data
+function handleConfirmDelete(data) {
+  confirmDeletePopup.open();
+  cardtoDeleteData = data;
+}
 
-const cardSection = new Section(initialCards, createCard, "#cards-list");
-const maxImagePopup = new PopupWithImage("#max-image-modal");
-const editProfilePopup = new PopupWithForm(
-  "#edit-profile-modal",
-  handleEditProfile
-);
-const addPlacePopup = new PopupWithForm("#add-card-modal", handleAddCardSubmit);
+//Deletes card
+function handleDeleteCard() {
+  api
+    .deleteCard(cardtoDeleteData._id)
+    .then(() => {
+      const card = document.querySelector(`li[id='${cardtoDeleteData._id}'`);
+      card.remove();
+    })
+    .catch((err) => {
+      console.log(err);
+    })
+    .finally(() => {
+      confirmDeletePopup.close();
+    });
+}
 
-const userInfo = new UserInfo({
-  nameSelector: ".profile__name",
-  jobSelector: ".profile__subtitle",
-});
+//Changes Profile Picture
+function handleEditPicSubmit({ link }) {
+  const submitButton = editProfilePicModal.querySelector(
+    "#edit-pic-modal-submit-button"
+  );
+  submitButton.textContent = "Saving...";
 
-//***************************************************************************************************************************//
-//                                             Validation and Rendering                                                      //
-//***************************************************************************************************************************//
+  api
+    .changeProfilePicture(link)
+    .then(() => {
+      profilePic.src = link;
+      profilePicPopup.close();
+    })
+    .catch((err) => {
+      console.log(err);
+    })
+    .finally(() => {
+      submitButton.textContent = "Save";
+    });
+}
 
-const editProfileFormValidator = new FormValidator(config, editProfilePopup.popupForm);
+//Handles card Like
+function handleLikeCard(card, method) {
+  if (method === "PUT") {
+    api
+      .likeCard(card._data._id)
```

Comment:
`_data`

Please, delete the low dash (`underscore`) from methods if you want to use them outside a class because the low dash shows a private method which is used only inside the class.